src/color-picker/color-picker.scss

```
@@ -0,0 +1,181 @@
+// Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+// See LICENSE.md in the project root for license terms and full copyright notice.
+@import '../style/index';
+@import '../menu/index';
+@import '../button/borderless';
+
+$iui-color-field-size: $iui-baseline * 19;
+$iui-color-swatch-box-shadow: inset 0 0 0 $iui-xxs - 1 
```

I'd say leave this inset box-shadow spread as 1px, it looks fine. 

Also weirdly, using a variable doesn't make sense here because it's just 1px

```suggestion
$iui-color-swatch-box-shadow: inset 0 0 0 1px 
```

---

Changed to 1px